app/models/organization.rb

```diff
@@ -107,6 +107,7 @@ def process_ssrs_parent
   end
 
   def update_ssr_org_name
+    self.org_children.each{ |org| org.update_ssr_org_name }
```

Comment:
`all_child_organizations` will fetch every descendant of `self`, while `org_children` only returns immediate children. @BigChiefSmidgeums which should it be?